src/components/classifier/NeedHelpButton.js

```diff
@@ -1,31 +1,40 @@
-import React from 'react'
+import React, { Component } from 'react'
 import {
     TouchableOpacity
 } from 'react-native'
 import DeviceInfo from 'react-native-device-info'
 import FontedText from '../common/FontedText'
 import PropTypes from 'prop-types'
 
-const NeedHelpButton = (props) => {
-    return (
-      <TouchableOpacity onPress={props.onPress}>
-        <FontedText style={styles.needHelpText}> 
-          NEED SOME HELP WITH THIS TASK?
-        </FontedText>
-      </TouchableOpacity>
-    )
+import * as colorModes from '../../actions/colorModes'
+
+export class NeedHelpButton extends Component {
```

Comment:
Another good catch :) I don't have a firm reason. I have a _mild_ preference for class components in React Native apps with dreams of community contribution because class components are more structurally similar to visual components in native Android and iOS apps, which broadens the pool of developers who don't have to code switch (no pun intended) to understand and modify our source code.